app/views/new_threads/show.html.erb

```diff
@@ -1,89 +1,79 @@
-<% @pageTitle = @new_thread.title + " | CAT Forum" %>
+<!-- <%= @pageTitle = @new_thread.title %>
 <% @PageDescription = @new_thread.description %>
-<% if user_signed_in? %>
-  <span class="user"> <%= current_user.username %></span>
-  <%= image_tag current_user.image.url(:thumb), size: "50x50", :class => 'user-img'%>
-<% end %>                                            
-<div class="thread-replies" itemscope itemtype="http://schema.org/Question">
-  <%= image_tag(@new_thread.user.image.url(:thumb), :class => "user-image") %>
-  <%= image_tag("arrow.png", class: "triangle") %>
-  <div class="speech-bubble">
+-->
```

Comment:
@ananyo2012  @sonalkr132 i am uncommenting the title and descriptions. This is again creating the same issue as I mentioned above. Please deploy and check. I could not come up with any alternate solution. I will keep looking for what is going wrong.